src/foundry/client/pixi/perception/vision-mode.d.mts

```diff
@@ -2,8 +2,6 @@ import type { AnyObject, InexactPartial, SimpleMerge, ValueOf } from "../../../.
 import type { DataField } from "../../../common/data/fields.d.mts";
 import type { fields } from "../../../common/data/module.d.mts";
 
-export {};
```

Comment:
Type check is complaining that `Namespace 'DataField' has no exported member 'Options'.`

Reply:
I saw that but since I didn't make any changes that I thought should've effected that, I ignored it.